addons/mail/static/src/core/common/store_service.js

```diff
@@ -336,6 +337,7 @@ export class Store extends BaseStore {
         this.ChatWindow.get(thread)?.update({ autofocus: 0 });
         this.env.services["discuss.rtc"].toggleCall(thread, { camera: true });
         this.openInviteThread = thread;
+        this.initialInviteThread = true;
```

Comment:
This cannot be a store global, if we need to track it this way it should at least be per thread.